torch/csrc/jit/ir.cpp

```diff
@@ -96,6 +96,64 @@ std::string escapeString(std::string s) {
   return s;
 }
 
+void printAttributeValue(std::ostream & out, Symbol & name, const Node * n) {
```

Comment:
Did the code move only, or are there changes?

Reply:
Just a move, this was needed in 2 places